vips/vip-Gateway/bsctestnet.ts

```diff
@@ -0,0 +1,45 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const COMPTROLLER_BEACON = "0xdddd7725c073105fb2abfcbdec16708fc4c24b74";
+export const VTOKEN_BEACON = "0xBF85A90673E61956f8c79b9150BAB7893b791bDd";
+export const NEW_COMPTROLLER_IMPLEMENTATION = "0x2Ba2fbFeBaA90eda758bc16f89DB6216387f4488";
+export const NEW_VTOKEN_IMPLEMENTATION = "0x344cD779C5aAF3436795B49f7C375E716A20f527";
+export const NATIVE_TOKEN_GATEWAY = "0xc2FF955724a4c07DdB13b65f4dA4Ca8B9c49E7c5";
+
```

Comment:
We would need to set the new MarketFacet in the Core pool. Also in the mainnet VIP

Reply:
We can wait to have more audits, to avoid re-work

Reply:
Yeah, we can do it for bsctestnet now.

Reply:
Done https://github.com/VenusProtocol/vips/pull/195/commits/7c5ed791622ec5b45acc6c6d09e104a3b58d88ec